### DISCUSSION TITLE
Is it possible to bundle a couple of components instead of them all with the WET?

### DISCUSSION THREAD

**Discussion by sgilbert-gouv** [wet-boew/wet-boew#9406] [General]
How hard would it be to just bundle the video player component so I can use it on a webpage? Like you know when you want to download jQuery UI, you can download exactly what you need instead of the all the components. Would it be easy to do the same thing the WET?

Thank you in advance :)

**Comment by ghost** [comment]
As far as I am aware nothing at the moment. You may be able to modify the grunt complier to achieve this.
